Source/Language/Flow/SetupSequencePhrase.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+
+namespace Moq.Language.Flow
+{
+	// keeping the fluent API separate from `SequenceMethodCall` saves us from having to
+	// define a generic variant `SequenceMethodCallReturn<TResult>`, which would be much more
+	// work that having a generic fluent API counterpart `SequenceMethodCall<TResult>`.
```

Comment:
This demonstrates a pattern that should probably be applied more in Moq. Moq's "plumbing" classes (e.g. `MethodCall`, `MethodCallReturn<TResult>`, `Mock`) should be made largely non-generic and kept strictly separate from the generic classes that are part of the fluent API. These latter types only flow the context of a fluent API "phrase" (i.e. a chain of method calls starting with `mock.Setup…` and ending with the end-of-line semicolon). That way, we could probably get rid of some code duplication.